tailwind.config.js

```diff
@@ -0,0 +1,27 @@
+/** @type {import('tailwindcss').Config} */
+export default {
+  content: [
+    "./index.html",
+    "./src/**/*.{js,ts,jsx,tsx}",
+  ],
+  theme: {
+    extend: {
+      colors: {
```

Comment:
Were these colors added due to the original implementation of layers? Or did you just want some additional custom colors that weren't available?

Reply:
These colors are used in our mock ups.